internal/node_loader.js

```diff
@@ -56,8 +58,20 @@ function resolveToModuleRoot(path) {
   return null;
 }
 
-function runfilesDir() {
-  return process.env.RUNFILES || process.env.TEST_SRCDIR;
+function loadRunfiles(manifestPath) {
+  return new Promise((resolve, reject) => {
+    const map = Object.create(null);
+    const input = fs.createReadStream(manifestPath, {encoding: 'utf-8'});
+    const manifest = readline.createInterface({input});
+
+    manifest.on('line', line => {
```

Comment:
what happens when reading fails here?

Reply:
Maybe this helps?

https://stackoverflow.com/questions/21771220/error-handling-with-node-js-streams

Reply:
I don't think that applies to the `manifest` object here, the return of `readline.createInterface` is not a stream. But I think I could listen for `error` events on the underlying stream?
It really needs unit tests, I don't know how to verify the error handling. Maybe it's time for `jasmine_node_test`